### ISSUE TITLE
create dual-unit altimeter (inHg + mbar)

### ISSUE THREAD

**Issue by tigert** [Juanvvc/c172p-detailed#447]
The altimeter could be re-done with a dual-window model which has both inches of mercury and also millibar in the same disc. For ease of calibrating the animations those discs could actually be separate objects in the 3d model, even though in the real unit they likely are painted on the same piece of rotating metal.

Here's a good resolution photo of mine, which we can use in the project, licensed with "whatever FOSS license you like": http://tigert.1g.fi/kuvat/Simulators/Flightgear/gauge-faces/aab.jpg/_full.jpg

**Comment by gilbertohasnofb** [comment]
Great idea :+1:

**Comment by thevirtualfer** [comment]
Tuomas, the set of photos you have are really, really, really and definitively really awesoooooome.

**Comment by tigert** [comment]
Yeah, happy if they are useful. They are from a Piper Archer and a Saratoga, I took them ages ago just for textures. Feel free to create gauges from them. Ideally something we could share with all FG aircraft, as many of those instruments are identical in many aircraft.

**Comment by gilbertohasnofb** [comment]
@tigert I tried opening the image you posted but your site seems to have expired :cry: would you mind sending me the file or uploading it using some service such as http://postimage.org/ ?

**Comment by gilbertohasnofb** [comment]
@tigert I have been looking at photographs of several Cessna 172's, particularly older models such as ours here, and I am yet to find one with a dual-unit altimeter. I understand this is handy, particularly if you are flying the plane in a place which would normally use mbar instead of inHg, but if this dual unit instrument is not a factory instrument then I am inclined to vote not to add it.

One possible solution would be to create two altimeters and add an option to toggle between them in the aircraft menu, something like:

```
-----------------------------
Altimeter unit
[X] inHg        [ ] mbar
-----------------------------
```

What do you guys think about this? @onox any opinions? If you think it's worth the effort, then I will create the new altimeter, but I am afraid it might not make it to our new release tomorrow.

**Comment by tigert** [comment]
Yeah, I think it is not common because most planes are located in either inHg or hectopascal areas and relatively few fly within both regions. 

But altimeters, like other "steam gauges" do not last forever, and they get replaced when they wear and break, so you could install one with dual scale just as fine, if you needed it - for example if you regularly fly to a country where the other units are being used. Most airliners have these, and with the new avionics, even smaller planes have it because it's easy to do in software (like the Garmin G1000 glass cockpit, it's a menu option to toggle) 

We do have a simulated plane that is easy to teleport to anywhere in the world, so I would argue it would make sense. Then it would also work in the case where you need to switch units mid flight - going to the menu in that case is not really realistic either :-)

**Comment by tigert** [comment]
I believer the altimeter image is this: 

http://koti.kapsi.fi/tigert/altimeter.jpg

**Comment by gilbertohasnofb** [comment]
I still don't like the idea of using one of those altimeters in our plane, as I said I haven't seen a single photograph of an older 172 having one of these (and of course I mean contemporary photographs, meaning that even if they got replaced, the owners replaced with the original instruments).

I really believe the best solution is to select the unit in the aircraft menu (just like you can select to have the GPS there). I would be happy to create the new instrument and implement the dialog if everyone agrees.
